arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -566,153 +572,2046 @@
+      "properties": {
+        "AccessedAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Last time a there was a receive request to this subscription."
+        },
+        "AutoDeleteOnIdle": {
+          "type": "string",
+          "format": "time-span",
+          "description": "TimeSpan idle interval after which the topic is automatically deleted. The minimum duration is 5 minutes."
+        },
+        "AvailabilityStatus ": {
+          "type": "string",
+          "description": "Entity availability status for the topic.",
+          "enum": [
+            "Available",
+            "Limited",
+            "Renaming",
+            "Restoring",
+            "Unknown"
+          ],
+          "x-ms-enum": {
+            "name": "NamespaceType",
+            "modelAsString": false
+          }
+        },
+        "CreatedAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Exact time the message was created."
+        },
+        "DefaultMessageTimeToLive": {
+          "type": "string",
+          "format": "time-span",
+          "description": "Default message time to live value. This is the duration after which the message expires, starting from when the message is sent to Service Bus. This is the default value used when TimeToLive is not set on a message itself."
+        },
+        "EnableBatchedOperations": {
+          "type": "boolean",
+          "description": "Value that indicates whether server-side batched operations are enabled.."
+        },
+        "EnableDeadLetteringOnFilterEvaluationExceptions": {
+          "type": "boolean",
+          "description": "Value that indicates if a subscription has dead letter support on Filter evaluation exceptions."
+        },
+        "EnableDeadLetteringOnMessageExpiration": {
+          "type": "boolean",
+          "description": "Value that indicates if a subscription has dead letter support when a message expires."
+        },
+        "ExtensionData": {
+          "type": "object",
+          "description": "Structure that contains extra data."
+        },
+        "ForwardDeadLetteredMessagesTo": {
+          "type": "string",
+          "description": "Path to the recipient to which the dead lettered messages are forwarded."
+        },
+        "ForwardTo": {
+          "type": "string",
+          "description": "Path to the recipient to which the dead lettered messages are forwarded."
+        },
+        "IsReadOnly": {
+          "type": "boolean",
+          "description": "Value that indicates whether the entity description is read-only."
+        },
+        "LockDuration": {
+          "type": "string",
+          "format": "time-span",
+          "description": "The lock duration time span for the subscription."
+        },
+        "MaxDeliveryCount": {
+          "format": "int32",
+          "type": "integer",
+          "description": "Number of maximum deliveries."
```

Comment:
What is the max value that the service supports? Can I say 10000000000000000 ? It would be nice if the description mentions that. Please add the "exclusiveMaximum" or "maximum" constraint.

Please take a look at the list of constraints that swagger supports and add them to the spec wherever applicable.
You can find them [here](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#items-object).